tests/snuba/api/endpoints/test_organization_events_stats_span_indexed.py

```diff
@@ -1319,6 +1319,66 @@ def test_device_class_filter(self):
             for test in zip(event_counts, rows):
                 assert test[1][1][0]["count"] == test[0]
 
+    def test_device_class_top_events(self):
+        event_counts = [
+            ("low", 6),
+            ("medium", 0),
+            ("low", 6),
+            ("medium", 6),
+            ("low", 0),
+            ("medium", 3),
+        ]
+        spans = []
+        for hour, count in enumerate(event_counts):
+            spans.extend(
+                [
+                    self.create_span(
+                        {
+                            "description": "foo",
+                            "sentry_tags": {
+                                "status": "success",
+                                "device.class": "1" if count[0] == "low" else "2",
```

Comment:
I just realized this only works because
- 1 is low
- 2 is medium
- 3 is high
If we ever update it to something like
- 1 and 2 is low
- 3 and 4 is low
- 5 and 6 is high
wouldnt this break? This is a potential future as that's the reason why it's being stored as these numbers instead of directly as the class

Reply:
updated the test to make it future proof